.env.example

```
@@ -90,6 +90,9 @@ POSTHOG_PROJECT_TOKEN=
 POSTHOG_HOST=
 # Content moderation model override (default: google/gemini-3.1-flash-lite)
 CONTENT_MODERATION_MODEL=
+# PII redaction model override (default: anthropic/claude-sonnet-5). Persist-time
+# scan that fails CLOSED, so recall is prioritized over cost.
+PII_REDACTION_MODEL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PII_REDACTION_MODEL key should go before the POSTHOG_HOST key

(UnorderedKey)